src/api/app/controllers/webui/package_controller.rb

```diff
@@ -179,18 +175,33 @@ def binary
   def binaries
     required_parameters :repository
     @repository = params[:repository]
-    begin
-      @buildresult = Buildresult.find_hashed(project: @project, package: @package,
-        repository: @repository, view: ['binarylist', 'status'])
-    rescue ActiveXML::Transport::Error => e
-      flash[:error] = e.message
-      redirect_back(fallback_location: { controller: :package, action: :show, project: @project, package: @package })
+
+    results_from_backend = Buildresult.find_hashed(project: @project, package: @package, repository: @repository, view: ['binarylist', 'status'])
+    unless results_from_backend
```

Comment:
You will not reach this is there is an `ActiveXML::Transport::Error`...

Reply:
Sure, if the `Buildresult.find_hashed` call fails then you got to the rescue part on line #202. The same as before...

Reply:
ah true